public/todo/AddTodo.js

```diff
@@ -2,14 +2,20 @@ import Component from '../Component.js';
 
 class AddTodo extends Component {
 
-    onRender(form) {
+    onRender(dom) {
         const onAdd = this.props.onAdd;
+        const form = dom.querySelector('form');
+        const input = dom.querySelector('input[name=todo]');
         
         form.addEventListener('submit', async event => {
             event.preventDefault();
 
```

Comment:
you need to grab the value of the input before you can pass it down to the object you're creating. 
Grab the value within the event listener. You could do something like 
`const newTodo = input.value`